cordapp-cpk2/src/test/kotlin/net/corda/plugins/cpk2/CpkExportReservedPackageNoAutoExportTest.kt

```diff
@@ -0,0 +1,56 @@
+package net.corda.plugins.cpk2
+
+import org.assertj.core.api.Assertions.assertThat
+import org.gradle.testkit.runner.TaskOutcome.FAILED
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.api.TestInstance.Lifecycle.PER_CLASS
+import org.junit.jupiter.api.TestReporter
+import org.junit.jupiter.api.io.TempDir
+import java.nio.file.Path
+
+@TestInstance(PER_CLASS)
+class CpkExportReservedPackageNoAutoExportTest {
+    private companion object {
+        private const val cordappVersion = "1.0.1-SNAPSHOT"
+    }
+
+    private lateinit var testProject: GradleProject
+
+    @BeforeAll
+    fun setup(@TempDir testProjectDir: Path, reporter: TestReporter) {
+        testProject = GradleProject(testProjectDir, reporter)
+            .withTestName("cpk-export-reserved-package")
+            .withSubResource("src/main/java/net/corda/contract/ExampleContract.java")
+            .withSubResource("src/main/java/net/corda/contract/package-info.java")
+            .withSubResource("src/main/java/net/corda/internal/Test.java")
```

Comment:
You should be able to add the exact same Java files into _both_ tests. The `net.cordapp.Test` file should be allowed inside a private package as well as an exported one, i.e. there shouldn't be any error concerning `net.cordapp` in the output.

Reply:
I've added `net.cordapp.Test` to the private test too